index.js

```diff
@@ -110,10 +109,9 @@ module.exports = function () {
 
         this.hostIsNested = this.hostEl.parentNode !== this.el;
 
-        // if the host el is nested one level deep we need a reference to it's container
+        // if the host el is nested we need a reference to it's container
         if (this.hostIsNested === true) {
             this.hostContainerEl = this.hostEl.parentNode;
```

Comment:
I'm seeing a couple of references to this.hostContainerEl in the event handlers. This worries me that there is still some logic tied up with our quick fix of just grabbing the immediate parent.

Reply:
After a bit more digging of my brain, I remember that for the case of our combobox where we overlaid an icon element on top of the textbox using CSS, we decided to set the clickTargetEl on the textbox and icon's containing element. But couldn't we just add `pointer-events: none` to the icon element and therefore remove this clickTargetEl logic?

Reply:
That would work, too. Let me look into that for our combobox.